lib/Basics/memory.cpp

```diff
@@ -21,12 +21,20 @@
 /// @author Dr. Frank Celler
 ////////////////////////////////////////////////////////////////////////////////
 
-#include "Basics/Common.h"
-
 #ifdef ARANGODB_ENABLE_MAINTAINER_MODE
 #include <cstring>
 #endif
 
+#include <cstdlib>
+
+#include "Basics/Common.h"
+#include "Basics/error.h"
+#include "Basics/voc-errors.h"
+
+#ifdef _WIN32
+#include <windows.h>
```

Comment:
do we really need windows.h?

Reply:
It is needed here for `SecureZeroMemory`.